ergoCubSN000/wrappers/inertials/alljoints-inertials_remapper.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+
+    <device xmlns:xi="http://www.w3.org/2001/XInclude" name="alljoints-inertials_remapper" type="multipleanalogsensorsremapper">
+        <param name="OrientationSensorsNames">
+            (head_imu_0 l_arm_ft l_leg_ft l_foot_ft r_arm_ft r_leg_ft r_foot_ft)
+        </param>
+        <action phase="startup" level="10" type="attach">
```

Comment:
We never completely aligned the all the attach/detach levels in all the configuration files, but I would try to at least stick to an agreed scheme at least for new files. Can we use 5 here as reported in https://github.com/robotology/robots-configuration/issues/377, or propose in https://github.com/robotology/robots-configuration/issues/377 a different scheme. That is indeed a rabbit hole of problems, so feel free to ignore for this PR.

Reply:
I could make this change in this PR, so we could align the files with the documentation :+1: